clayground.py

```diff
@@ -0,0 +1,130 @@
+import lancedb
+import matplotlib.pyplot as plt
+import rasterio as rio
+import streamlit as st
+from rasterio.plot import show
+
+st.set_page_config(layout="wide")
+
+
+# Get preferrred chips
+def get_unique_chips(tbl):
+    chips = [
+        {"tile": "17MNP", "idx": "0271", "year": 2023},
+        {"tile": "19HGU", "idx": "0033", "year": 2018},
+        {"tile": "33NVB", "idx": "0393", "year": 2020},
+        {"tile": "21JVJ", "idx": "0100", "year": 2020},
+        {"tile": "34KHD", "idx": "0080", "year": 2018},
+        {"tile": "19JCF", "idx": "0215", "year": 2023},
+        {"tile": "20HMK", "idx": "0100", "year": 2020},
+        {"tile": "37MFT", "idx": "0313", "year": 2023},
+        {"tile": "49KHR", "idx": "0020", "year": 2017},
+        {"tile": "55LBC", "idx": "0075", "year": 2022},
+    ]
+
+    filter = " OR ".join(
+        [
+            f"(tile == '{chip['tile']}' AND idx == '{chip['idx']}') AND year == {chip['year']}"
+            for chip in chips
+        ]
+    )
```

Comment:
Just need to wrap L27 to <88 characters to make the linter happy.

Reply:
By the way @srmsoumya, [`filter`](https://docs.python.org/3/library/functions.html#filter) is a built-in Python function. Can we change the variable name to something else?

Reply:
Ok, I just went ahead and renamed `filter` to `tile_filter` at commit f4731c63186723a26563364de9994f42a08ee6a7